rusoto/core/src/signature.rs

```diff
@@ -55,46 +66,57 @@ impl SignedRequest {
             canonical_uri: String::new(),
         }
     }
+    /// Adds a header of format <"content-type", content-type>
     pub fn set_content_type(&mut self, content_type: String) {
         self.add_header("content-type", &content_type);
     }
 
+    /// Sets the target hostname
     pub fn set_hostname(&mut self, hostname: Option<String>) {
         self.hostname = hostname;
     }
 
+    /// Sets the target hostname using the current service type and region
```

Comment:
Perhaps something pointing to how it builds the hostname, incase you're trying to troubleshoot this method?

```rust
/// Sets the target hostname based on the current service type, and region.
///
/// See the implementation of build_hostname to see how this is done.
```